src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        class="form__search"
+        action="#"
+        method="post"
+        data-qa="big"
+      >
+        <input
+          class="big__input input__bar"
+          type="search"
+          name="bigSearch"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          required
+        >
+        <img
+          class="big__icon icon"
+          src="./images/Search.svg"
+          alt="icon"
+        >
```

Comment:
Why do you put an icon after input? 
It would be more logical to put the icon first, like on the design. 
It will not confuse other developers and will reduce the needed indents for elements in CSS